<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>trackviewer</class>
 <widget class="QMainWindow" name="trackviewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Track Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>848</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_file"/>
    <addaction name="actionOpen_3D_points"/>
   </widget>
   <widget class="QMenu" name="menuView_Mode">
    <property name="title">
     <string>View Mode</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView_Mode"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionOpen_3D_points">
   <property name="text">
    <string>Open 3D points</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
